org.osgi.test.junit5/test.bndrun

```diff
@@ -24,6 +24,9 @@
 	org.osgi.service.http.port=*,\
 	org.osgi.framework.bootdelegation=sun.reflect,\
 	osgi.console=
+#	,\
+#	tester.names=org.osgi.test.junit5.service.ServiceExtension_SanityCheckingTest
```

Comment:
Do we really want to commit this? This seems like something you used to test your changes?

Reply:
Err, no. I was clearly in too much of a hurry to commit. :smile: Sorry. 

The idea of that line was so it would only run the test i was working on, speeds up development.